compiler/src/main/scala/org/scalajs/core/compiler/PrepJSInterop.scala

```diff
@@ -693,7 +694,7 @@ abstract class PrepJSInterop extends plugins.PluginComponent
         }
 
         val ownerLoadSpec = jsInterop.jsNativeLoadSpecOf(sym.owner)
-        val jsName = jsInterop.jsNameOf(sym)
+        val JSName.Literal(jsName) = jsInterop.jsNameOf(sym)
```

Comment:
Other question: How are you going to implement the desugaring of `LoadJSConstructor` for a class like this? Or do we not want to support it anymore for these classes (but how to remain backwards compatible)?